Storage/rest/advanced_hypermedia.h

```diff
@@ -34,15 +34,21 @@ SOFTWARE.
 #include "hypermedia.h"
 #include "sfinae.h"
 
-CURRENT_STRUCT(AdvancedHypermediaRESTRecordResponse) {
+namespace current {
+namespace storage {
+namespace rest {
+
+CURRENT_STRUCT_T(AdvancedHypermediaRESTRecordResponse) {
+  CURRENT_FIELD(success, Optional<bool>);
```

Comment:
@sompylasar `null` won't be printed by the JSON serializer used for Hypermedia.
`Optional` is effectively the easiest way to hint the JSON builder to skip this field unless it's set.